var/spack/repos/builtin/packages/henson/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Henson(CMakePackage):
+    """Cooperative multitasking for in situ processing."""
+
+    homepage = "https://github.com/henson-insitu/henson"
+    url      = "https://github.com/henson-insitu/henson"
+    git      = "https://github.com/henson-insitu/henson.git"
+
+    version('master', branch='master')
```

Comment:
is this your development branch? If so, you can use the spack reserved version name `develop` here :)

Reply:
It's really the only version right now. So I guess it counts as the development version. I'll rename it to `develop`.